components/splitter/accessibility/overview.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Accessibility Overview
+page_title: Telerik UI for Blazor Splitter Documentation | Splitter Accessibility Overview
+description: "Get started with the Telerik UI for Blazor Splitter and learn about its accessibility support for WAI-ARIA, Section 508, and WCAG 2.2."
+tags: telerik,blazor,accessibility,wai-aria,wcag,splitter
+slug: splitter-accessibility-overview
+position: 0
+---
+
+# Accessibility Overview
+
+The UI for Blazor Splitter component is [WCAG 2.2 AA](https://www.w3.org/TR/WCAG22) and [Section 508](https://www.section508.gov) compliant. The component also follows the [WAI-ARIA best practices](https://www.w3.org/WAI/ARIA/apg/) for implementing the keyboard navigation for its component [role](https://www.w3.org/TR/wai-aria/#roles), and is tested against the popular screen readers.
+
+# Blazor Splitter Accessibility Example
+
+WCAG 2.2 introduces the ["Dragging Movements"](https://www.w3.org/WAI/WCAG22/Understanding/dragging-movements) criterion as an important part of the Operable principle. Its primary goal is to guarantee that any feature reliant on drag actions offers an alternative method that can be executed with a single click, enhancing user accessibility.
+
+In our illustrative example below, we've showcased the pane resize action, achievable through our [Context Menu]({%slug contextmenu-overview%}). Our goal is to offer a versatile API that allows users to trigger all functions programmatically or externally, meeting diverse accessibility requirements for any applications.
```

Comment:
"We" statements are not recommended. See [here ](https://docs.telerik.com/style-guide/basic-rules-and-guidelines#pronouns) for reference. 

Consider an alternative approach:

```suggestion
The illustrative example below shows the pane resize action, achievable through the [Context Menu]({%slug contextmenu-overview%}). Telerik UI for Blazor aims to offer a versatile API that allows users to trigger all functions programmatically or externally, meeting diverse accessibility requirements for any applications.
```